src/analyses/spec.ml

```diff
@@ -284,11 +284,6 @@ struct
     (* try to evaluate the expression using query
        -> if the result is the same as tv, do the corresponding transition, otherwise remove the entry from the domain
        for pointers this won't help since it always returns `Top *)
-    ( let i = ctx.ask (Queries.EvalInt exp) in (* when (Queries.ID.is_bool i) *)
-      (match Queries.ID.to_bool i with
-      | Some b when b<>tv -> M.debug "EvalInt: `Int bool" (* D.remove k m TODO where to get the key?? *)
-      | _ -> M.debug "EvalInt: `Int no bool")
-    );
```

Comment:
This is indeed weird code, but the comment above is about this I guess. Although seems like they don't agree.

Reply:
I now removed the comment as well.